src/components/AboutSectionBucket.vue

```diff
@@ -34,8 +34,26 @@
   </div>
 </template>
 
-<script>
-export default {
-  props: ['data']
-};
+<script lang="ts">
+
+interface Buckets{
+  title_h3: string;
+  description: string;
+}
+
+interface Data{
+  title_h2: string;
+  bucket: Buckets;
+}
```

Comment:
For individual components have a common file from were you export **interface** using models
reference: https://dev.to/jwp/typescript-models-and-javascript-objects-31b3